From 11a93604f21b0e7f95d6ed114779ccac91637fb9 Mon Sep 17 00:00:00 2001
From: Chris Dimich <chris.dimich@boundarydevices.com>
Date: Tue, 8 Mar 2022 19:05:07 -0800
Subject: [PATCH 1/1] start_isp.sh: remove 4k and -lm option

4k is not supported on boundary kernel and causes unexpected issues.
Also remove uneeded -lm option from run.sh script, which also causes
issues.

Signed-off-by: Chris Dimich <chris.dimich@boundarydevices.com>
---
 imx/start_isp.sh | 17 +++--------------
 1 file changed, 3 insertions(+), 14 deletions(-)

diff --git a/imx/start_isp.sh b/imx/start_isp.sh
index 8a07acfed..962876115 100755
--- a/imx/start_isp.sh
+++ b/imx/start_isp.sh
@@ -16,20 +16,9 @@ if [ $NR_DEVICE_TREE_BASLER -eq 1 ]; then
 	echo "Starting isp_media_server for Single Basler daA3840-30mc"
 
 	cd $RUNTIME_DIR
-
-	if [ $NR_DEVICE_TREE_OV5640 -eq 0 ]; then
-
-		# Default configuration for Basler daA3840-30mc: basler_4k
-		# Available configurations: basler_4k, basler_1080p60, basler_4khdr, basler_1080p60hdr
-		exec ./run.sh -c basler_4k -lm
-
-	elif [ $NR_DEVICE_TREE_OV5640 -eq 1 ]; then
-
-		# Default configuration for Basler daA3840-30mc: basler_1080p60
-		# Available configurations: basler_1080p60, basler_1080p60hdr
-		exec ./run.sh -c basler_1080p60 -lm
-
-	fi
+	# Default configuration for Basler daA3840-30mc: basler_1080p60
+	# Available configurations: basler_1080p60, basler_1080p60hdr
+	exec ./run.sh -c basler_1080p60
 
 elif [ $NR_DEVICE_TREE_BASLER -eq 2 ]; then
 
-- 
2.31.1

